src/software/ai/navigator/path_planner/no_path_test_path_planner.h

```diff
@@ -14,11 +14,13 @@ class NoPathTestPathPlanner : public PathPlanner
      *
      * @param start start point
      * @param destination destination point
-     * @param field  WARNING: NoPathTestPathPlanner does not respect the field
+     * @param navigableArea WARNING: NoPathTestPathPlanner does not respect the
+     * navigableArea
      * @param obstacles WARNING: NoPathTestPathPlanner does not avoid obstacles
      *
      * @return a vector that is {}
      */
-    PathType findPath(const Point &start, const Point &destination, const Field &field,
-                      const std::vector<Obstacle> &obstacles) override;
+    Path findPath(const Point &start, const Point &destination,
+                  const Rectangle &navigableArea,
```

Comment:
`navigableArea` -> `navigable_area` to be consistent with variable declarations in the rest of the codebase

Reply:
done

Reply:
Doesn't look like this changed?

Reply:
ok done

Reply:
looks like there are still a few instances of `navigableArea` in this PR, could we switch them all over?

Reply:
done